app/main/appium-method-handler.js

```diff
@@ -70,7 +70,10 @@ export default class AppiumMethodHandler {
     // Give the source/screenshot time to change before taking the screenshot
     await Bluebird.delay(500);
 
-    let sourceAndScreenshot = await this._getSourceAndScreenshot();
+    let sourceAndScreenshot;
```

Comment:
I assume an empty arrayshould be assigned to this variable since we apply ellipsis operator to it later

Reply:
If the ellipsis operator has no affect on an object if it's `undefined` or `null`.